go/api/response_types.go

```diff
@@ -146,3 +151,61 @@ func CreateEmptyClusterValue() ClusterValue[interface{}] {
 		value: Result[interface{}]{val: empty, isNil: true},
 	}
 }
+
+type XPendingSummary struct {
+	NumOfMessages    int64
+	StartId          Result[string]
+	EndId            Result[string]
+	ConsumerMessages Result[[]ConsumerPendingMessage]
```

Comment:
```suggestion
	ConsumerMessages []ConsumerPendingMessage
```
Array in go is `nil`able, so you can skip additional wrapping

Reply:
Done